exec.go

```diff
@@ -41,42 +53,70 @@ var execCommand = cli.Command{
 	},
 }
 
-func execProcess(context *cli.Context, config *specs.Process) (int, error) {
+func execProcess(context *cli.Context, args []string) (int, error) {
 	container, err := getContainer(context)
 	if err != nil {
 		return -1, err
 	}
-	process := newProcess(*config)
 
+	bundle := container.Config().Rootfs
+	if err := os.Chdir(path.Dir(bundle)); err != nil {
+		return -1, err
+	}
+	spec, _, err := loadSpec(specConfig, runtimeConfig)
+	if err != nil {
+		return -1, err
+	}
+
+	// override the cwd, if passed
+	if context.String("cwd") != "" {
+		spec.Process.Cwd = context.String("cwd")
```

Comment:
I see what you are doing here but it would be confusing for future readers.  Maybe assign the process to a var and not use it directly off of the spec or add a comment.  Both solutions would be best.  Something along the lines of `using the spec's main process as a template for the new process`.

What do you think?